docs/guide/http_cache.md

```diff
@@ -695,55 +714,55 @@ parameters:
     ezplatform.http_cache.tags.header_reduced_ttl: 7200
 ```
 
-### Response tagging done with Content View
+### Response tagging done with content view
 
-For Content views response tagging is done for you, and cache system will output headers like the following:
+For content views response tagging is done automatically, and cache system outputs headers like the following:
 
 ```
 HTTP/1.1 200 OK
 Cache-Control: public, max-age=86400
 xkey: ez-all c1 ct1 l2 pl1 p1 p2
 ```
 
-If the given content have several Locations you'll see several `l<location-id>` and `p<location-id>` tags in the response.
+If the given content has several Locations, you can see several `l<location-id>` and `p<location-id>` tags in the response.
 
 !!! note "How response tagging for ContentView is done internally"
 
     In `ezplatform-http-cache` there is a dedicated response listener `HttpCacheResponseSubscriber` that checks if:
     
-    - Response has attribute `view`
-    - View implements `eZ\Publish\Core\MVC\Symfony\View\CachableView`
-    - Cache is not disabled on the individual view
+    - the response has attribute `view`
+    - the view implements `eZ\Publish\Core\MVC\Symfony\View\CachableView`
+    - cache is not disabled on the individual view
 
-    If that checks out, the response will be adapted with the following:
+    If that checks out, the response is adapted with the following:
     
-    - `ResponseCacheConfigurator` will apply site access settings for enabled/disabled cache and default TTL
-    - `DispatcherTagger` will dispatch the built in ResponseTaggers which will generate the tags as described above.
+    - `ResponseCacheConfigurator` applies SiteAccess settings for enabled/disabled cache and default TTL
+    - `DispatcherTagger` dispatches the built-in ResponseTaggers which generate the tags as described above.
 
-    Further information can be found in ezplatform-http-cache's `docs/response_taggers.md`.
+    Further information can be found in [`ezplatform-http-cache` docs.](https://github.com/ezsystems/ezplatform-http-cache/blob/master/docs/response_taggers.md)
 
-### Response tagging in Controllers
+### Response tagging in controllers
 
 For tagging needs in controllers, there are several options, here presented in recommended order:
 
-1\. Reusing DispatcherTagger to get it to pick correct tags for you.
+1\. Reusing `DispatcherTagger` to pick correct tags.
 
-Examples for tagging everything needed for Content using autowirable `ResponseTagger` interface:
+Examples for tagging everything needed for content using the autowirable `ResponseTagger` interface:
```

Comment:
Does it mean "Examples for using the autowirable `ResponseTagger` interface to tag everything that is needed for the content:" ?